Hippo.Web/ClientApp/src/util/StringHelpers.ts

```diff
@@ -7,3 +9,17 @@ export const SplitCamelCase = (str: string) => {
       .replace(/^./, (str) => str.toUpperCase())
   );
 };
+
+export const getGroupModelString = (arg: GroupModel | GroupModel[]) => {
+  const groups = Array.isArray(arg) ? arg : [arg];
+  var values = [
+    // start with names of groups for sortability
+    ...groups.map((g) => g.name),
+    // then include remaining properties for filtering
+    ...groups.map((g) => [
+      g.displayName,
+      g.admins.map((a) => [a.name, a.email, a.kerberos]),
```

Comment:
seems like it'd be a little confusing to me if you type in the admin's name or email and the row filtered, unless it displayed the admin in the actual cell, which i don't think it does.

Reply:
It displays admin name and email in the tooltip.